wp-content/themes/ipbs-largo/images/map.svg

```diff
@@ -0,0 +1,963 @@
+				</g>
+			</g>
+			<g class="coverage">
+				<g>
+					<circle opacity="0.7" fill="#245A8F" cx="723.2" cy="198.5" r="142"/>
+					<text class="tv" x="961" y="182">WNIT - TV</text>
+
+					<line opacity="0.7" fill="none" stroke="#245A8F" stroke-width="3" stroke-miterlimit="10"
+						  x1="861.3" y1="176.5" x2="951.3" y2="176.5"/>
+				</g>
+				<g>
+					<circle opacity="0.3" fill="#ED2224" cx="741.4" cy="197.6" r="100"/>
+					<text class="fm" x="961" y="204">WVPE - TV</text>
+
+					<line opacity="0.3" fill="none" stroke="#ED2224" stroke-width="3" stroke-miterlimit="10"
+						  x1="841.3" y1="197.5" x2="951.3" y2="197.5"/>
+				</g>
+			</g>
+			<g class="label">
+				<g>
+					<text x="613.2" y="183.4">South Bend/Elkhart</text>
+				</g>
+				<circle fill="#989898" cx="718.7" cy="199.3" r="14"/>
+				<!-- <circle opacity="0.7" fill="#245A8F" cx="723.2" cy="198.5" r="5"/>
+				<circle opacity="0.3" fill="#ED2224" cx="741.4" cy="197.6" r="5"/> -->
+				<rect x="629" y="169.9" opacity="1.000000e-02" fill="#FFFFFF" width="213" height="40.7"
+					  onclick="animateClickedCity('south_bend_x2F__elkhart')"/>
+			</g>
+		</g>
+		<g id="fort_wayne">
+			<g class="heading">
+				<g>
+					<text x="1050" y="222">Fort Wayne</text>
+				</g>
+			</g>
+			<g class="coverage">
+				<g>
+
+					<line opacity="0.7" fill="none" stroke="#245A8F" stroke-width="3" stroke-miterlimit="10"
+						  x1="991.3" y1="241.5" x2="1044.3" y2="241.5"/>
+					<text class="tv" x="1050" y="248">WFWA - TV</text>
+					<circle opacity="0.7" fill="#245A8F" cx="890.8" cy="306" r="122.5"/>
+				</g>
+				<g>
+
+					<line opacity="0.3" fill="none" stroke="#ED2224" stroke-width="3" stroke-miterlimit="10"
+						  x1="883.3" y1="261.5" x2="1044.3" y2="261.5"/>
+					<text class="fm" x="1050" y="269">WCKZ - FM</text>
+					<circle opacity="0.3" fill="#ED2224" cx="852.8" cy="211" r="60.3"/>
+				</g>
+				<g>
+					<circle opacity="0.3" fill="#ED2224" cx="899.4" cy="307.6" r="100"/>
+
+					<line opacity="0.3" fill="none" stroke="#ED2224" stroke-width="3" stroke-miterlimit="10"
+						  x1="994.3" y1="283.5" x2="1044.3" y2="283.5"/>
+					<text class="fm" x="1050" y="292">WBOI - FM</text>
+				</g>
+				<g>
+
+					<line opacity="0.3" fill="none" stroke="#ED2224" stroke-width="3" stroke-miterlimit="10"
+						  x1="891.3" y1="308.5" x2="1044.3" y2="308.5"/>
+					<text class="fm" x="1050" y="314">W204BF - FM</text>
+					<circle opacity="0.3" fill="#ED2224" cx="877.4" cy="323.6" r="22.7"/>
+				</g>
+				<g>
+					<line opacity="0.3" fill="none" stroke="#ED2224" stroke-width="3" stroke-miterlimit="10"
+						  x1="927.3" y1="329.5" x2="1044.3" y2="329.5"/>
+					<text class="fm" x="1050" y="337">WBNI - FM</text>
+					<circle opacity="0.3" fill="#ED2224" cx="881.4" cy="332.1" r="46.5"/>
+				</g>
+			</g>
+			<g class="label">
+				<g>
+					<text x="849.2" y="293.4">Fort Wayne</text>
+					<path fill="#515350" d="M859.4,286.h8.2v2.5h-5.2v2.6h4.4v2.5h-4.4v3h5.4v2.5h-8.4V286.4z"/>
```

Comment:
Safari reports:
> Error: Problem parsing `d="M859.4,286.h8.2v2.5h-5.2v2.6h4.4v2.5h-4.4v3h5.4v2.5h-8.4V286.4z"`

But it also says this on ipbs.org, so I'm not sure that this is a cause of the "Can't find Velocity" problem in this PR.

Reply:
The "Can't find Velocity" error is fixed in https://github.com/INN/umbrella-ipbs/commit/b790eadcf3505bfb42f457c67c5bacc70d149519 and it looks like all the parts of Fort Wayne are displaying, so I'm not sure what the problem is.

![Screen Shot 2020-01-27 at 15 14 41](https://user-images.githubusercontent.com/1754187/73210202-c39b2780-4117-11ea-9f9a-164083cce32e.png)